app/config/LoginConfig.scala

```diff
@@ -20,12 +21,17 @@ case class LoginConfig(
 )
 
 object LoginConfig {
- def forStage(stageOpt: Option[String]): LoginConfig = {
+  def forStage(stageOpt: Option[String]): LoginConfig = {
     val stage = stageOpt.getOrElse("DEV")
     val domain = stage match {
-      case "PROD" => "gutools.co.uk"
       case "DEV" => "local.dev-gutools.co.uk"
-      case x => x.toLowerCase() + ".dev-gutools.co.uk"
+      case "CODE" => "code.dev-gutools.co.uk"
```

Comment:
do we need to consider `TEST`?

Reply:
No - login.gutools is never deployed to TEST so that isn't a valid value for this config (and if we ever did decide to say replace CODE with TEST we'd need to do a proper read through the config to generate the right value at that point)